src/components/Login/Login.js

```diff
@@ -34,6 +33,8 @@ class Login extends Component {
         this.setState({ loaded: true });
       } else if (result) {
         UserUtil.getMyProfile(this.onTokenValidCheck.bind(this));
+      } else {
+        this.setState({ loaded: true });
```

Comment:
I understand there is some case that both result and error can be null, but it seems kind of weird and potentially dangerous. Is this can be fixed from the server? or do you guys have any idea about this